### DISCUSSION TITLE
2023-09-05

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#475] [daily_meeting]
### 오늘의 할 일
1. 개인 미션 진행(백 제출 월,  프 제출 목)
2. 사용자 유치 계획 (9/8)
3. 기존 개인용 서비스를 유지하는 방향에 대해 논의

### 개인 일정
노아 - 근로 5시

**Comment by woosung1223** [comment]
### 체크인
7점 - 컨디션은 나쁘지 않지만 조금 피곤 하네여 

### 체크아웃
6점 - 오늘 삽질만 한거 같아서 슬픕니다 ㅠㅠ

**Comment by aak2075** [comment]
### 체크인

10점 - 잘 자서 컨디션 좋아요 오늘도 화이팅

### 체크아웃

10점 - 리팩토링 재밌는데 적당히 끝내고 다음 단계 해야할 것 같아요

**Comment by nlom0218** [comment]
### 체크인

9점 - 벌써 배고프네요 😭

### 체크아웃

8점 - 재밌게 최적화를 하고 있는데 비가 많이 오네요 참..

**Comment by woo-jk** [comment]
### 체크인

8점 - 요즘 잠을 너무 늦게 자게된다

### 체크아웃

9점 - 성능 미션 재밌따

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 어제 못잔 잠을 오늘 보충해서 좋네요. 오랜만에 러닝도 했는데 덕분에 더 개운한 것 같습니다. 얼른 강의 끝내고 밥먹으러 가고 싶네요~
### 체크아웃
8점 - 삽질 많이해서 더 내려갈 뻔 했으나 마지막에 좀 감을 잡은 것 같아서 1점만 깎았읍니다..

**Comment by yeopto** [comment]
### 체크인

10점 - 새롭게 태어나서 그런지 피곤해도 기분이 좋네요.

### 체크아웃

10점 - 오늘도 뿌듯하게 지나간거 같습니다.

**Comment by jaehee329** [comment]
### 체크인
5점 - 갈수록 자도 피곤한데요?
### 체크아웃
6점 - 테코톡 발표자료가 거의 만들어졌어요
